docs/mp/tracing/01_tracing.adoc

```diff
@@ -68,6 +115,718 @@ To disables MP Tracing as by specification:
 mp.opentracing.server.skip-pattern=.*
 ----
 
+==== Configuration using MP Config
+
+Tracing configuration can be defined in `microprofile-config.properties` file.
+
+[source, properties]
+.Tracing configuration
+----
+tracing.components.web-server.spans.0.name="HTTP Request"
+tracing.components.web-server.spans.0.logs.0.name="content-write"
+tracing.components.web-server.spans.0.logs.0.enabled=false
+----
+
+==== Path based configuration in Helidon Web Server
+
+For Web Server we have a path based support for configuring tracing, in addition
+to the configuration described above.
+
+Configuration of path can use any path string supported by the
+Web Server. The configuration itself has the same possibilities
+as traced configuration described above. The path specific configuration
+will be merged with global configuration (path is the "newer" configuration, global is the "older")
+
+[source,properties]
+.Configuration properties
+----
+tracing.paths.0.path="/favicon.ico"
+tracing.paths.0.enabled=false
+tracing.paths.1.path="/metrics"
+tracing.paths.1.enabled=false
+tracing.paths.2.path="/health"
+tracing.paths.2.enabled=false
+----
+
+==== Renaming top level span using request properties
+
+To have a nicer overview in search pane of a tracer, you can customize the top-level span name using configuration.
+
+Example:
+[source,properties]
+.Configuration properties
+----
+tracing.components.web-server.spans.0.name="HTTP Request"
+tracing.components.web-server.spans.0.new-name: "HTTP %1$s %2$s"
+----
+
+This is supported ONLY for the span named "HTTP Request" on component "web-server".
+
+Parameters provided:
+
+1. Method - HTTP method
+2. Path - path of the request (such as '/greet')
+3. Query - query of the request (may be null)
+
+
+TODO section
+
+== Examples
+
+
+== Getting Started with Tracing
+
+The examples in this guide demonstrate how to integrate tracing with Helidon, how to view traces, how to trace
+across multiple services, and how to integrate tracing with Kubernetes.  All examples use Zipkin and traces
+will be viewed using both the Zipkin API and UI.
```

Comment:
I seem to recall that as a project we wanted to emphasize Jaeger and not Zipkin. If so we should adjust the example accordingly.

Reply:
If you don't mind we could do it in another iteration?

Reply:
Fair enough.